### PR TITLE
docs(0020): add Interledger: Explain Like I'm Five (ELI5)

### PR DESCRIPTION
**Pull request by emschwartz**
Replaces https://github.com/interledger/rfcs/pull/193

@professorhantzen I squashed your PR into one commit and gave it the RFC number 20. If this looks good to you we can merge this in

### CODE REVIEW

**Comment by peterjkrey** [comment]
Reviewing now

**Comment by emschwartz** [comment]
Nice work @professorhantzen and hooray for your first contribution! 🎉 🎉 🎉

**Comment by professorhantzen** [comment]
Thank you @emschwartz , it's really an honour.

@peterjkrey Thanks for the review! I agree about the diagrams and am working on getting some done.
